### DISCUSSION TITLE
Code problem in the request song action

### DISCUSSION THREAD

**Discussion by fsan2val** [AzuraCast/AzuraCast#6567] [Ideas]
When you request songs more than 2 times (you make a request and then another) a red message appears accompanied by a badly elaborated code at the beginning of the notice, which causes a bad appearance and confusion and of course it is not trustworthy.

I was able to test in its demo, it still has the same error.

I hope this helps

![AzuraTest-Radio-AzuraCast](https://github.com/AzuraCast/AzuraCast/assets/69484466/6b053e49-0bc2-4a52-8010-df1cb188a166)

![AzuraTest-Radio-AzuraCast 3rror](https://github.com/AzuraCast/AzuraCast/assets/69484466/f52236cc-c0d7-4a3f-86b6-ee298eb3358f)

**Comment by Moonbase59** [comment]
See https://github.com/AzuraCast/AzuraCast/issues/6568.

**Reply by fsan2val** [reply]
I hope they fix this soon. Thanks for listening as always. greetings

**Comment by BusterNeece** [comment]
Improved in the latest Rolling Release.
